rclcpp/include/rclcpp/allocator/allocator_deleter.hpp

```diff
@@ -94,11 +94,11 @@ void set_allocator_for_deleter(AllocatorDeleter<T> * deleter, Alloc * alloc)
 
 template<typename Alloc, typename T>
 using Deleter = typename std::conditional<
-    std::is_same<typename std::allocator_traits<Alloc>::template rebind_alloc<T>,
-    typename std::allocator<void>::template rebind<T>::other>::value,
-    std::default_delete<T>,
-    AllocatorDeleter<Alloc>
-    >::type;
+  std::is_same<typename std::allocator_traits<Alloc>::template rebind_alloc<T>,
+  typename std::allocator<void>::template rebind<T>::other>::value,
+  std::default_delete<T>,
+  AllocatorDeleter<Alloc>
+  >::type;
```

Comment:
This is better. It's unfortunate that the last line `>::type` isn't indented one less (should end on the same column as the line that starts it).

Reply:
A similar comment applies to all follow multi-line templates.

Reply:
When targeting the latest release this "improvement" is sadly not possible. Therefore I revert this change.